ast/src/lib.rs

```diff
@@ -72,6 +72,7 @@ pub fn evaluate_unary_operation<T: FieldElement>(op: UnaryOperator, v: T) -> T {
         UnaryOperator::Plus => v,
         UnaryOperator::Minus => -v,
         UnaryOperator::LogicalNot => v.is_zero().into(),
+        UnaryOperator::Next => panic!("Cannot evaluate \"'\"."),
```

Comment:
I think it could be fine to actually return `v` here, but I'm leaving it like that for now.